app/static/js/script.js

```diff
@@ -1,33 +1,49 @@
+import $ from "https://cdn.skypack.dev/jquery@3.6.0";
+//Accessiblity Feature - Dark/HighvContrast Mode
 document.addEventListener('DOMContentLoaded', function () {
   var modeSwitch = document.querySelector('.mode-switch');
+});
 
-  modeSwitch.addEventListener('click', function () {                     document.documentElement.classList.toggle('dark');
-    modeSwitch.classList.toggle('active');
+function confirmLogOutIntent() {
+  $('a#logoutModal').click(function () {
+    if (confirm('Are you sure to logout')) {
+      return true;
+    }
+    return false;
   });
-  
-  var listView = document.querySelector('.list-view');
-  var gridView = document.querySelector('.grid-view');
-  var projectsList = document.querySelector('.project-boxes');
-  
-  listView.addEventListener('click', function () {
-    gridView.classList.remove('active');
-    listView.classList.add('active');
-    projectsList.classList.remove('jsGridView');
-    projectsList.classList.add('jsListView');
-  });
-  
-  gridView.addEventListener('click', function () {
-    gridView.classList.add('active');
-    listView.classList.remove('active');
-    projectsList.classList.remove('jsListView');
-    projectsList.classList.add('jsGridView');
-  });
-  
-  document.querySelector('.messages-btn').addEventListener('click', function () {
-    document.querySelector('.messages-section').classList.add('show');
-  });
-  
-  document.querySelector('.messages-close').addEventListener('click', function() {
-    document.querySelector('.messages-section').classList.remove('show');
-  });
-});
\ No newline at end of file
+}
+
+modeSwitch.addEventListener('click', function () {
+  document.documentElement.classList.toggle('dark'); modeSwitch.classList.toggle('active');
```

Comment:
Codacy has a fix for the issue: [Replace `'dark');·modeSwitch.classList.toggle('active'` with `"dark");⏎··modeSwitch.classList.toggle("active"`](https://app.codacy.com/gh/Terry-BrooksJr/animated-chainsaw/pullRequest?prid=9263623)
```suggestion
  document.documentElement.classList.toggle("dark");
```